src/objects/Body.ts

```diff
@@ -246,6 +248,7 @@ export class Body extends EventTarget {
     this.aabbNeedsUpdate = true
     this.boundingRadius = 0
     this.wlambda = new Vec3()
+    this.sensor = options.sensor ?? false
```

Comment:
it should be a boolean only

```suggestion
    this.sensor = Boolean(options.sensor)
```